### PR TITLE
Updating the interactive tutorials in Russian without Katacoda

### LINKED ISSUE #41923: [ru] Remove Katacoda tutorials and update instructions
Umbrella issue: #41496

**Problem:**
Outdated (i.e. non-working) Katacoda-related content is still presented in Russian localisation.

**Proposed Solution:**
We must remove it following relevant changes in the English version (#40306).

**Page to Update:**
* https://kubernetes.io/ru/docs/tutorials/hello-minikube/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/scale/scale-intro/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/scale/scale-interactive/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/explore/explore-intro/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/explore/explore-interactive/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/expose/expose-intro/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/expose/expose-interactive/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/update/update-intro/
* https://kubernetes.io/ru/docs/tutorials/kubernetes-basics/update/update-interactive/
* https://kubernetes.io/ru/docs/contribute/style/style-guide/

/area localization
/language ru
/triage accepted

### PR DESCRIPTION
**Pull request by shurup**
Fixes #41923

All tutorials where Katacoda was in use are now synchronised with their actual English editions. I also had to make a general overhaul for these pages fixing many things and applying consistent wording for terms, etc.

@Arhell @kirkonru: I kindly ask you to find some time to go through this ASAP. **Anyway, I want this to go live in a week**. These updates are essential for users, and rendering them took a lot of time, so I don't want to postpone them much more. We'll be better able to fix any issues later.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

File: content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
 
-            <p>В предыдущем модуле мы промасштабировали приложение до нескольких экземпляров. Это необходимо сделать, чтобы иметь возможность обновлять приложение, не влияя на его доступность. По умолчанию максимальное количество подов, которое может быть недоступно во время обновления, и максимальное количество новых подов, которое можно создать, равны 1. Эти две опции могут быть определены в абсолютном (числа) или относительном соотношении (проценты).
-            В Kubernetes обновления версионируются, поэтому любое обновление развёртывания можно откатить до предыдущей (стабильной) версии.</p>
+            <p>Пользователи надеются, что приложения будут работать круглосуточно, а разработчики в свою очередь хотят развёртывать новые версии приложений по несколько раз в день. В Kubernetes это возможно благодаря механизму плавающих обновлений (rolling updates). <b>Плавающие обновления</b> позволяют обновить деплойменты без простоев, шаг за шагом заменяя старые поды на новые. Новые поды будут запущены на узлах, имеющих достаточно ресурсов.</p>
```
Comment (kirkonru) [inline content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html:30]:
```suggestion
            <p>Пользователи ожидают, что приложения будут работать круглосуточно и бесперебойно, а разработчикам часто приходится развёртывать новые версии приложений по несколько раз в день. Механизм плавающих обновлений (rolling updates) Kubernetes примирияет эти взаимоусключающие устремления. <b>Плавающие обновления</b> позволяют обновить деплойменты без простоев, постепенно заменяя старые поды на новые. При этом новые поды будут запускаться на узлах с достаточными ресурсами.</p>
```

File: content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 2]
```diff
@@ -94,20 +94,20 @@ <h2 style="color: #3771e3;">Обзор плавающих обновлений</
...
+                    <li> откатываться к предыдущим версиям;</li>
+                    <li> осуществлять непрерывную интеграцию и непрерывную доставку приложений без простоев.</li>
 
                 </ul>
 
             </div>
             <div class="col-md-4">
                 <div class="content__box content__box_fill">
-                    <p><i>Если развёртывание было открыто наружу, в процессе обновления сервис будет балансировать нагрузку трафика только на доступные поды.</i></p>
+                    <p><i>Если деплоймент был открыт наружу, в процессе обновления сервис будет балансировать нагрузку трафика только на доступные поды.</i></p>
```
Comment (kirkonru) [inline content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html:110]:
```suggestion
                    <p><i>Если деплоймент был открыт наружу, в процессе обновления сервис будет балансировать трафик только на доступные поды.</i></p>
```

File: content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 3]
```diff
@@ -116,16 +116,75 @@ <h2 style="color: #3771e3;">Обзор плавающих обновлений</
...
+               <h3>Обновление версии приложения</h3>
+               <p>Чтобы получить список деплойментов, выполните подкоманду <code>get deployments</code>:
+               <code><b>kubectl get deployments</b></code></p>
+               <p>Для списка подов — подкоманду <code>get pods</code>:</p>
+               <p><code><b>kubectl get pods</b></code></p>
+               <p>Чтобы увидеть версию текущего образа в приложении, воспользуйтесь подкомандой <code>describe pods</code> и посмотрите на поле <code>Image</code>:</p>
+               <p><code><b>kubectl describe pods</b></code></p>
+               <p>Чтобы обновить версию образа приложения до v2, воспользуемся подкомандой <code>set image</code>, для которой укажем имя деплоймента и новую версию нужного образа:</p>
+               <p><code><b>kubectl set image deployments/kubernetes-bootcamp kubernetes-bootcamp=jocatalin/kubernetes-bootcamp:v2</b></code></p>
+               <p>Эта команда перевела деплоймент на использование другого образа для приложения и инициировала плавающее обновление. Можно проверить статус подов: новых и старых, работа которых прекращается, — с помощью подкоманды <code>get pods</code>:</p>
```
Comment (kirkonru) [inline content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html:135]:
```suggestion
               <p>Эта команда перевела деплоймент на использование другого образа для приложения и инициировала плавающее обновление. Статус новых и старых подов (т. е. тех, которые будут остановлены) можно проверить с помощью подкоманды <code>get pods</code>:</p>
```

File: content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 4]
```diff
@@ -116,16 +116,75 @@ <h2 style="color: #3771e3;">Обзор плавающих обновлений</
...
+               <p><code><b>kubectl get pods</b></code></p>
             </div>
         </div>
+        <div class="row">
+            <div class="col-md-12">
+               <h3>Шаг 2: валидация обновления</h3>
+               <p>Во-первых, убедимся, что приложение работает. Доступный вовне IP-адрес и порт узнаем с помощью команды <code>describe service</code>:</p>
```
Comment (kirkonru) [inline content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html:142]:
```suggestion
               <p>Во-первых, убедимся, что приложение работает. Доступный извне IP-адрес и порт узнаем с помощью команды <code>describe service</code>:</p>
```

File: content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 5]
```diff
@@ -116,16 +116,75 @@ <h2 style="color: #3771e3;">Обзор плавающих обновлений</
...
+               <p><code><b>kubectl get pods</b></code></p>
+               <p>Здесь обратите внимание, что некоторые поды перешли в статус <tt>ImagePullBackOff</tt>.</p>
+               <p>Чтобы получить больше информации о проблеме, выполните подкоманду <code>describe pods</code>:</p>
+               <p><code><b>kubectl describe pods</b></code></p>
+               <p>В разделе <code>Events</code> вывода по проблемным подам можно увидеть, что новая версия образа (<code>v10</code>) не существует в репозитории.</p>
+               <p>Чтобы откатить деплоймент к последней работающей версии, воспользуйтесь подкомандой <code>rollout undo</code>:</p>
+               <p><code><b>kubectl rollout undo deployments/kubernetes-bootcamp</b></code></p>
+               <p>Команда <code>rollout undo</code> откатывает деплоймент к предыдущему известному состоянию (к образу v2). Обновления версионируются, поэтому можно откатиться к любому предыдущему известному состоянию деплоймента.</p>
+               <p>Вызовем подкоманду <code>get pods</code>, чтобы снова вывести список подов:</p>
```
Comment (kirkonru) [inline content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html:174]:
```suggestion
               <p>С помощью подкоманды <code>get pods</code> выведем список подов еще раз:</p>
```

File: content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 6]
```diff
@@ -116,16 +116,75 @@ <h2 style="color: #3771e3;">Обзор плавающих обновлений</
...
+               <p><code><b>kubectl get pods</b></code></p>
+               <p>Четыре пода работают. Чтобы проверить, какие версии образа развёрнуты в этих подах, выполните подкоманду <code>describe pods</code>:</p>
```
Comment (kirkonru) [inline content/ru/docs/tutorials/kubernetes-basics/update/update-intro.html:176]:
```suggestion
               <p>Четыре пода работают. Проверить, какие версии образа развёрнуты в этих подах, можно подкомандой <code>describe pods</code>:</p>
```

**Review by kirkonru — APPROVED** [review]

**Review by kirkonru — APPROVED** [review]

**Comment by shurup** [comment]
Oh, I've just squashed everything :)
Thanks, @kirkonru, for all corrections! We are ready to go now.

/lgtm
/approve

**Pull request merged**